src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -352,6 +352,12 @@ Plugin::Plugin()
           [&](const Config& config) {
               return _metrics->GetGops(get_specified_device_name(config));
           }}},
+        {ov::execution_devices.name(),
+         {true,
+          ov::PropertyMutability::RO,
+          [&](const Config& config) {
+              return std::string("NPU" + config.get<DEVICE_ID>());
```

Comment:
device id is added via "." symbols, e.g. `NPU.1`

Reply:
updated now, thanks!